package.json

```diff
@@ -13,14 +13,14 @@
     "node": ">=16.18.0"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test",
-    "test:watch": "tsdx test --watch",
-    "test:e2e": "tsdx test --config jest-e2e-config.json",
+    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build --incremental'",
+    "build": "tsc --project ./tsconfig.build.json",
```

Comment:
Currently, this will only build a `CommonJS` module. But we should also target an `ESM` module. We could use a similar approach to here https://github.com/wagmi-dev/viem/blob/main/package.json#L9-L11 and add additional build commands. These would make use of the CLI flags `--module` and `--outDir`

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/382/commits/f8f57164632dc2b38ae659018942bef31458b0d8